lib/rubocop/cop/rails/schema_comment.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for migrations using `add_column` without passing a
+      # comment option.
+      #
+      # @example
+      #   # bad (no comment for a new column)
+      #   add_column :table, :column, :integer
+      #
+      #   create_table :table do |t|
+      #     t.type :column
+      #   end
+      #
+      #   # good
+      #   add_column :table, :column, :integer, comment: 'Number of offenses'
+      #
+      #   create_table :table do |t|
+      #     t.type :column, comment: 'Number of offenses'
+      #   end
+      #
+      class SchemaComment < Base
+        include ActiveRecordMigrationsHelper
+
+        COLUMN_MSG = 'New database column without `comment`.'
+        TABLE_MSG = 'New database table without `comment`.'
+        RESTRICT_ON_SEND = %i[add_column create_table].freeze
+        CREATE_TABLE_COLUMN_METHODS = %w[
+          :column
+          :integer
+          :string
+          :text
+          :float
+          :decimal
+          :time
+          :date
+          :datetime
+          :timestamp
+          :binary
+          :boolean
```

Comment:
Missing `:blob`, and `:numeric`? 

https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column

(I'm wondering if there's a way to write this cop without having to explicitly list all the types).

Reply:
It could be a block_list of the other methods which are less, but that could lead to a worst bug where the cop would raise and offense for a new `remove` or `change`  interface.